pkgs/development/tools/mold/default.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv
```

Comment:
mold no longer relies on an external `tbb`. `tbb` is now included as a subdirectory of the project and statically-linked to mold executable.

Reply:
Fixed, ty.